aten/src/ATen/core/dispatch/Dispatcher.h

```diff
@@ -369,7 +369,7 @@ class TORCH_API Dispatcher final {
  * to lookup a kernel for a certain set of arguments.
  */
 class TORCH_API OperatorHandle {
-  template <typename T> friend struct std::hash;
```

Comment:
Err, but it is indeed a struct, isn't it https://en.cppreference.com/w/cpp/utility/hash